datafusion/physical-plan/src/aggregates/row_hash.rs

```diff
@@ -100,22 +100,24 @@ struct SpillState {
 ///
 /// See "partial aggregation" discussion on [`GroupedHashAggregateStream`]
 struct SkipAggregationProbe {
-    /// Number of processed input rows
+    /// Number of processed input rows (updated during probing)
     input_rows: usize,
-    /// Number of total group values for `input_rows`
+    /// Number of total group values for `input_rows` (updated during probing)
     num_groups: usize,
 
-    /// Aggregation ratio check should be performed only when the
-    /// number of input rows exceeds this threshold
+    /// Aggregation ratio check performed when the number of input rows exceeds
+    /// this threshold (from `SessionConfig`)
     probe_rows_threshold: usize,
-    /// Maximum allowed value of `input_rows` / `num_groups` to
-    /// continue aggregation
+    /// Maximum ratio of `input_rows` / `num_groups` to continue aggregation
```

Comment:
The ratio seems to be stated inversely here?

```suggestion
    /// Maximum ratio of `num_groups` to `input_rows` for continuing aggregation
```
My understanding is based on the following code:

```rust
            self.should_skip = self.num_groups as f64 / self.input_rows as f64
                >= self.probe_ratio_threshold;
```

Reply:
Yes, I agree -- thank you for the catch. @korowa is that your understanding too?

Reply:
Yes, the numerator and denominator in this comment should be swapped (I've double checked `Config`, they are correct). Thank you!